Userland/Libraries/LibGUI/Scrollbar.cpp

```diff
@@ -19,56 +19,28 @@ REGISTER_WIDGET(GUI, Scrollbar)
 
 namespace GUI {
 
-static constexpr Gfx::CharacterBitmap s_up_arrow_bitmap {
-    "         "
-    "         "
-    "         "
-    "    #    "
-    "   ###   "
-    "  #####  "
-    " ####### "
-    "         "
-    "         ",
-    9, 9
+static const AK::Vector<Gfx::IntPoint, 3> s_up_arrow_coords {
```

Comment:
These static arrays don't need to by `Vectors`, just make them `constexpr Arrays`.